pypose/module/lqr.py

```diff
@@ -345,10 +344,11 @@ def lqr_backward(self, x_init, dt, u_traj=None, u_lower=None, u_upper=None, du=N
             Qxx, Qxu = Qt[..., :ns, :ns], Qt[..., :ns, ns:]
             Qux, Quu = Qt[..., ns:, :ns], Qt[..., ns:, ns:]
             qx, qu = qt[..., :ns], qt[..., ns:]
-            Quu_inv = torch.linalg.pinv(Quu)
 
-            K[...,t,:,:] = Kt = - Quu_inv @ Qux
-            k[...,t,:] = kt = - bmv(Quu_inv, qu)
+            K[...,t,:,:] = Kt = -torch.linalg.lstsq(Quu, Qux).solution
+            k[...,t,:] = kt = -torch.linalg.lstsq(Quu, qu.unsqueeze(-1)).\
+                solution.squeeze(-1)
```

Comment:
How about 
```python
from torch.linalg import lstsq
```
then
```python
K[...,t,:,:] = Kt = -lstsq(Quu, Qux).solution
k[...,t,:] = kt = -lstsq(Quu, qu.unsqueeze(-1)).solution.squeeze(-1)
```

Reply:
Thanks for your advice. I've pushed the updated version.